examples/content_types/src/main.rs

```diff
@@ -27,7 +33,10 @@ struct Person {
 #[get("/<name>/<age>", format = "json")]
 fn get_hello(name: String, age: u8) -> Json<String> {
     // NOTE: In a real application, we'd use `rocket_contrib::json::Json`.
-    let person = Person { name: name, age: age, };
+    let person = Person {
+        name: name,
+        age: age,
+    };
```

Comment:
These lines are far too short for `rustfmt` to be trying to split them!

Reply:
Well, I agree but it also ease updates reducing merge conflicts down the road. Having those inlined will generate merge conflict in case more than one person is adding / removing fields.

Reply:
What I am more worried about with these lines is that a future release of rustfmt will want to format this differently (i.e. merge the lines back together because it is short)

Reply:
Well, in that case, we will be able to change the settings I guess. Also, why do you think it's gonna change soon ?

Reply:
In this specific case rustfmt has the option `struct_lit_single_line`, which should fit the literal on one line. Since the default for that setting is `true`, this change looks like a bug in rustfmt. I am against enforcing style changes that are due to bugs, especially when the knobs that control those formatting options don't have any stability guarantees.

More generally, rustfmt makes *many* changes that appear to be due to bugs or surprising heuristics for "small" that actually make code (IMO) less readable, and even configuring a bunch of unstable options doesn't lead to correct results.

Reply:
Fair enough :)
Thanks for explanations !